README.md

```diff
@@ -504,32 +333,17 @@ client = AerieClient(AerieHostSession(
 
 If you'd like to contribute to this project, you'll first need to clone this repository, and you will have to install [`poetry`](https://python-poetry.org/docs/master/).
 
-Then, you will need to run the following commands:
-
-1. `poetry install` -- installs the necessary dependencies into a poetry-managed virtual environment.
-2. `poetry run pre-commit install` -- creates a git [pre-commit](https://pre-commit.com) hook which will automatically run formatters, style checks, etc. against your proposed commits.
-
-### Deployment
-
-To deploy a new version of aerie-cli:
-
-- Verify the version number in `pyproject.toml` is incremented from the current version in PyPI
-- Clear out any previous build artifacts, if present, from the root of the repository:
-  ```sh
-  rm -rf dist/
-  ```
-- Build the project using Poetry:
-  ```sh
-  poetry build
-  ```
-
-To upload to PyPI, use twine:
+From the root of the repo, use `poetry` to install Aerie-CLI and its dependencies:
 
 ```sh
-python3 -m twine upload dist/*
+poetry install
 ```
 
-Details can be found in the [twine documentation](https://twine.readthedocs.io/en/stable/index.html).
+To run commands from source as you edit, use the `poetry run` command. For example:
+
+```
+poetry run aerie-cli plans simulate --id 42
+```
```

Comment:
Interesting. I use `pip install -e .` and then run `aerie-cli <steps>`

Reply:
That works too. If you install with poetry, I think you need to use `poetry run` (but I also use Pip editable mode). I plan to update these docs whenever I get around to sorting out the formatter and pre-commit hook so that everybody's dev environment is fairly consistent. (I might even move away from Poetry if it doesn't seem to be adding value...)

Reply:
I think if we're both using pip editable mode then to keep any future dev env consistent, the docs oughta give the pip -e instructions.

Reply:
We might need to move to poetry in order to get the pre-commit hook working; I know some folks are set up with poetry already.

Reply:
(So, I plan to get in line with the docs in the near future, either by changing the docs to match my dev env or by changing my env to match the docs.)